solana/pyth2wormhole/client/src/main.rs

```diff
@@ -826,6 +840,7 @@ async fn attestation_job(args: AttestationJobArgs) -> Result<(), ErrBoxSend> {
         batch_no, batch_count, group_name, seqno
     );
     ATTESTATIONS_OK_CNT.inc();
+    HEALTHCHECK_STATE.lock().await.add_result(true); // Report this job as successful to healthcheck
```

Comment:
Can you move it the line after? Since it's a blocking operation it's better than there'd be nothing after it. 

In this case the gauge will be more accurate if it is before this statement.